Assignment2-Ancestors/Q1.py

```diff
@@ -0,0 +1,44 @@
+from Tree import BinaryTree
+
+def find_ancestors(tree, key):
+    """
+    Find ancestors of the first node with the given value while preorder traversal and save the Nodes in a list that return.
+    :param tree: Tree
+    :param key: any
+    :return: list(Nodes)
+    """
+    node = tree.find(key)
+    if node:

```

Comment:
I understand you, thanks) 
But still is it better to raise an error when node is not found or should we let the function return None?